extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/GraphqlMethodsProcessor.java

```diff
@@ -10,21 +10,23 @@
 import io.quarkus.deployment.annotations.BuildStep;
 import io.quarkus.deployment.execannotations.ExecutionModelAnnotationsAllowedBuildItem;
 import io.smallrye.graphql.api.Subscription;
+import io.smallrye.graphql.api.federation.Resolver;
 
 public class GraphqlMethodsProcessor {
     private static final DotName QUERY = DotName.createSimple(Query.class);
     private static final DotName MUTATION = DotName.createSimple(Mutation.class);
     private static final DotName SUBSCRIPTION = DotName.createSimple(Subscription.class);
+    private static final DotName RESOLVER = DotName.createSimple(Resolver.class);
 
     @BuildStep
     ExecutionModelAnnotationsAllowedBuildItem graphqlMethods() {
         return new ExecutionModelAnnotationsAllowedBuildItem(new Predicate<MethodInfo>() {
             @Override
             public boolean test(MethodInfo method) {
-                // maybe just look for `@GraphQLApi` on the declaring class?
```

Comment:
The annotation I added is not annotated with `@GraphQLApi` thus why I removed the comment.

It might make sense to add it? If so, we could keep the comment.